src/ServiceControl/Licensing/LicenseController.cs

```diff
@@ -52,6 +53,8 @@ public class LicenseInfo
             public string InstanceName { get; set; }
 
             public string LicenseStatus { get; set; }
+
+            public string[] ConnectedApplications { get; set; }
```

Comment:
Should this be part of the license information or maybe part of the root controller instead? This is a runtime information and for now does not influence the type/content of the license.

Reply:
I would say it should be a persisted collection because who knows what we are going to do with it in the future ;-)